### PR TITLE
Adding files so German translation becomes visible

### PR DESCRIPTION
**Pull request by Nick-Tallguy**
These 3 files will make the de section all visible, but they need translating themselves. The quick & dirty way is to accept the pull request, then translate the files. For the period of time involved the de home page will be in English.

### CODE REVIEW

**Comment by geomantic** [comment]
Thanks for taking a look at this, @Nick-Tallguy ! 

^^ @sfkeller, now that Nick has helped with the fix, please give it a try. Thanks!

**Comment by Nick-Tallguy** [comment]
There is one dead link from the German home page - basically you have;  
http://teachosm.org/en/intro/whyteach/ and you also need  
http://teachosm.org/de/intro/whyteach/  which doesn't exist yet. 

Until that file is created, the moving door link on the German front page will lead to a 404 error.

This is a spreadsheet I maintain for keeping track of modules on LearnOSM - https://docs.google.com/spreadsheets/d/1NlJHTYAYpyyF5o9D8UMAsSzFG7lHY2cIi2mzncx530w/edit?usp=sharing

Basically if you have a module in English, to prevent broken links you need a similar module, or what I've called an 'interim' module, in every other language - it works the other way as well - if you've got it in German you need it in English to avoid 404's. This is an example of what I've called an 'interim' module - http://learnosm.org/it/beginner/id-editor/

Good luck & give me a shout if I can help on anything else (sorry about the delay that can be involved though!).

**Comment by sfkeller** [comment]
Thanks! (Backlinking to #92)
